cmd/libs/go2idl/client-gen/testdata/apis/testgroup/v1/register.go

```diff
@@ -40,7 +40,10 @@ func addKnownTypes(scheme *runtime.Scheme) {
 	)
 
 	scheme.AddKnownTypes(SchemeGroupVersion,
-		&v1.ListOptions{})
+		&v1.ListOptions{},
```

Comment:
We'll move this into something like "pkg/apimachinery/server/types/v1/registered.go#AddToScheme(...)" eventually - so that you declare which API group you support but don't describe the types yourself.